server/config/urls.py

```diff
@@ -18,4 +19,5 @@
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
+    url(r'^index/', TemplateView.as_view(template_name='index.html')),
```

Comment:
@nirgn975 - yes, but these url-s do not include a view for index.html, which is in the title of this issue... should I check another url?